webapi/scripts/Readme.md

```diff
@@ -8,33 +8,33 @@ This script will create the ``knora-test-unit`` repository and not load any data
 
  - with GraphDB running inside a docker container:
  ```$ graphdb-se-docker-init-knora-test-unit.sh```
- 
+
  - with GraphDB running locally, e.g., graphdb distribution:
  ```$ graphdb-se-local-init-knora-test-unit.sh ```
- 
- 
-### Initialize for using in the test / staging environment 
 
-This script will create the ``knora-test`` repository and load data specified in ``graphdb-knora-test-data.ttl``.
+
+### Initialize for using in the test / staging environment
+
+This script will create the ``knora-test`` repository and load test data specified in ``graphdb-knora-test-data.expect``.
 
  - with GraphDB running inside a docker container:
  ```$ graphdb-se-docker-init-knora-test.sh```
- 
+
  - with GraphDB running locally, e.g., graphdb distribution:
  ```$ graphdb-se-local-init-knora-test.sh ```
- 
 
-### Initialize for using in the production environment 
+
+### Initialize for using in the production environment
 
 This script will create the ``knora-prod`` repository and load data specified in ``graphdb-se-knora-prod-data.ttl``.
 
  - with GraphDB running inside a docker container:
  ```$ graphdb-se-docker-init-knora-prod.sh```
- 
+
  - with GraphDB running locally, e.g., graphdb distribution:
  ```$ graphdb-se-local-init-knora-prod.sh ```
- 
+
 
 ## Fuseki
 
-Nothing neede to do. Simply run Fuseki, either in a docker container or locally by using the supplied distribution.
\ No newline at end of file
+Nothing neede to do. Simply run Fuseki, either in a docker container or locally by using the supplied distribution.
```

Comment:
Do we still support Fuseki?

Reply:
typo "neede"

Reply:
All the Fuseki support is still there, it's just deprecated, like the docs say: http://www.knora.org/documentation/manual/rst/knora-api-server/deployment/webapi-getting-started.html?highlight=deprecated